docs/developing-fuzzbench/custom_analysis_and_reports.md

```diff
@@ -39,7 +39,7 @@ experiment](https://www.fuzzbench.com/reports/sample/index.html) like this:
 ```bash
 mkdir ~/my-report; cd ~/my-report
 wget https://www.fuzzbench.com/reports/sample/data.csv.gz
-PYTHONPATH=<fuzzbench_root> python3 experiment/generate_report.py \
+POSTGRES_PASSWORD=1 PYTHONPATH=<fuzzbench_root> python3 analysis/generate_report.py \
```

Comment:
Please remove POSTGRES_PASSWORD=1, this should be fixed with https://github.com/google/fuzzbench/pull/346. Thanks for letting us know of the bug.

Reply:
Thanks for flagging @vanhauser-thc and thanks for fixing @inferno-chromium